test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase_SUCCESS.lua

```diff
@@ -114,6 +128,24 @@ local function setChoiseSet(choiceIDValue)
   return temp
 end
 
+--! @setChoiceSet: Creates Choice structure without VRcommands
```

Comment:
@setChoiceSet must be matched to function name -> @setChoiceSet_noVR

Reply:
Updated!